### PR TITLE
Resprites kitchen appliances, Adds kitchen particle smoke

### PR DESCRIPTION
**Pull request by atteria**
The stove, oven, and fryer have had their sprites improved. No more weird perspective, weird shading, and now the fry baskets can be visually removed.
Stoves and ovens now emit smoke particles when food is about to burn, which gets more extreme as they get closer to burning.
The stove's active animation was broken, and never appeared. It's been replaced with an overlay. Sadly it can't layer over lighting, or it would layer over pots and pans too.
The oven's on animation is now an overlay that plays regardless of if the door is opening. It also layers over lighting.

kitchen showcase

https://user-images.githubusercontent.com/86520558/230571299-097c85b5-844f-4d89-a6ba-182d117a2b9f.mp4

### CODE REVIEW

**Comment by atteria** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by niennab — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/cooking/machinery/cooking_machines/oven.dm [hunk 1]
```diff
@@ -51,12 +51,13 @@
...
-			icon_state = "ovenclosed_on"
-		else
-			icon_state = "ovenclosed_off"
+		icon_state = "ovenclosed"
 	else
 		icon_state = "ovenopen"
+	cut_overlays()
+	if (!stat)
+		var/glow = image('icons/obj/cooking_machines.dmi', "oven_on", EFFECTS_ABOVE_LIGHTING_LAYER)
+		add_overlay(glow)
```
Comment (Generalcamo) [inline code/modules/cooking/machinery/cooking_machines/oven.dm:60]:
```suggestion
		add_screen_overlay(glow)
```

File: code/modules/cooking/machinery/cooking_machines/stove.dm [hunk 2]
```diff
@@ -30,6 +30,9 @@
 /obj/machinery/appliance/cooker/stove/update_icon()
 	. = ..()
 	cut_overlays()
+	if (!stat)
+		var/glow = image('icons/obj/cooking_machines.dmi', "stove_burner")
+		add_overlay(glow)
```
Comment (Generalcamo) [inline code/modules/cooking/machinery/cooking_machines/stove.dm:35]:
```suggestion
		add_screen_overlay(glow)
```

File: code/modules/cooking/machinery/cooking_machines/oven.dm (hunk 1 above)
Comment (atteria) [reply code/modules/cooking/machinery/cooking_machines/oven.dm:60]: It looks like `add_screen_overlay()` is only defined on `/obj/machinery/vending/`.

Comment (atteria) [reply code/modules/cooking/machinery/cooking_machines/oven.dm:60]: `make_screen_overlay()` also doesn't accept an `image()`
Testing it, there's too much bloom, and it causes the heating element glow to layer over cookware.

**Pull request merged**